src/sys/socket/addr.rs

```diff
@@ -920,6 +920,15 @@ impl From<SockaddrIn> for net::SocketAddrV4 {
 }
 
 #[cfg(feature = "net")]
+#[allow(clippy::from_over_into)]
```

Comment:
Would you like to do:

```
impl From<libc::sockaddr_in> for SockaddrIn
impl From<SockaddrIn> for libc::sockaddrin

impl From<libc::sockaddr_in6> for SockaddrIn6
impl From<SockaddrIn6> for libc::sockaddrin6
```

So that we can do roundtrips and also make clippy happy:)

Reply:
Done.